src/main/java/hudson/plugins/locale/LocaleFilter.java

```diff
@@ -59,6 +76,7 @@ public Locale nextElement() {
                         };
                     }
                 };
+                ((HttpServletResponse) response).addHeader("X-Jenkins-Language", locale.toString());
```

Comment:
If I understood correctly it's only used by test to know which language was set?